src/components/Calendar/Calendar.jsx

```diff
@@ -41,7 +41,7 @@ import * as Styled from './styles';
 const Calendar = () => {
   const defaultDate = new Date();
   const [currentDate, setCurrentDate] = useState(defaultDate);
-  const [selectedDate, setSelectedDate] = useState(null);
+  const [selectedDate, setSelectedDate] = useState([]);
```

Comment:
You store an array in `selectedDate` variables. Could it be more relevant to name this variable `selectedDates`?